2018-October-30/writingSamples/heslin.tex

```diff
@@ -0,0 +1,170 @@
+Therefore, to develop new therapies for these deficits, there is a
+\emph{critical need} to investigate the neural circuitry underlying
+cognitive symptoms of schizophrenia.
+
+The cerebellum is largely associated with coordinating motor functions,
+however, relayed cerebellar projections to cortical areas like the
+prefrontal cortex are instrumental in a variety of cognitive and
+executive functions\textsuperscript{7,8}. In line with these
+observations, one neuronal pathway implicated in the cognitive symptoms
+of schizophrenia is a cerebello-thalamo-cortico- cerebellar
+loop\textsuperscript{9}. Decreased structural and functional
+connectivity within this pathway and dysfunction of its nodes has been
+observed in patients with schizophrenia\textsuperscript{10--12}. It is
+hypothesized that the role of the cerebellum in this circuit is to
+coordinate cognitive processing through modulation of areas like the
+prefrontal cortex\textsuperscript{13,14}. The dentate nuclei, or lateral
+cerebellar nuclei (LCN) in rodents, send the majority of cerebellar
+input to the prefrontal cortex\textsuperscript{15,16}. A population of
+inhibitory D1 dopamine receptor (D1DR)-expressing neurons in the LCN has
+been recently implicated in pre-frontal dependent cognitive tasks
+\textsuperscript{17}. However, precisely how D1DR-expressing cells of
+the LCN contribute to cognitive function and modulation of the frontal
+cortex is unclear.
+
+To address this, we will evaluate the role of LCN D1DR-expressing
+neurons in performance of an interval timing task. In interval timing
+tasks, subjects are presented with a cue that signals them to respond
+after a certain duration elapses (e.g., press a lever after 12 seconds).
+Interval timing performance is impaired in schizophrenia
+patients\textsuperscript{18--21}, depends on multiple nodes in our
+circuit of interest\textsuperscript{22,23}, and is impacted by
+manipulations of dopaminergic signaling\textsuperscript{24--26}.
+Successful interval timing performance requires recruitment and
+coordination of several different executive functions such as working
+memory, attention, and decision making. Importantly, interval timing
+task measures can be readily compared across animal models,
+neuropsychiatric patients, and healthy human comparisons. Therefore, it
+is an ideal type of task with which to probe our circuit and neuronal
+population of interest in rodent models. Specifically, we will
+investigate how LCN D1DR-expressing neurons contribute to performance in
+a peak interval timing task, and how performance correlates with
+population activity in the LCN and prelimbic cortex (PL), the rodent
+medial frontal cortex. Based on our prior work and the primary
+literature, our \textbf{\emph{overall hypothesis}} is that inhibitory
+D1DR-expressing cells of the LCN are necessary for successful interval
+timing through modulation of activity in the PL.
+
+\textbf{\emph{Aim 1. Determine if LCN D1DR-expressing neurons are
```

Comment:
Identify, elucidate, describe the relationship... translate the ifs to hows.

concept of why it's necessary versus sufficient